src/test/rbd_mirror/test_mock_ImageSyncThrottler.cc

```diff
@@ -100,6 +100,7 @@ std::vector<MockImageSync *> MockImageSync::instances;
 
 
 // template definitions
+#include "tools/rbd_mirror/InstanceSyncThrottler.cc"
```

Comment:
A template specialization of InstanceSyncThrottle should be defined above and eliminate the need to pull in the CC file here. That would also allow the test cases below to specifically test ImageSyncThrottler instead of its interactions w/ InstanceSyncThrottler.